utils/rebuild.js

```diff
@@ -0,0 +1,157 @@
+/**
+ * JavaScript
+ * utils/rebuild.js
+ * 
+ * Provides functionality to rebuild the db from
+ * a given event
+ */
+
+const { db } = require('../src/connect')();
+
+const calcPb = require('../src/event-input/calcPb');
+const checkRecords = require('../src/event-input/checkRecords');
+
+let upToEvent = process.argv[2];
+if (!upToEvent) throw new Error('no event given');
+try {
+    upToEvent = parseInt(upToEvent);
+} catch (error) {
+    throw new Error('no event given');
```

Comment:
This error condition happens when the input is invalid, I think. Perhaps instead:

```suggestion
    throw new Error('invalid event number given');
```